src/app/shared/menu/menu-item/link-menu-item.component.html

```diff
@@ -3,6 +3,8 @@
    [attr.aria-disabled]="!hasLink"
    [title]="item.text"
    [routerLink]="getRouterLink()"
+   (click)="$event.stopPropagation()"
```

Comment:
Due to the `[routerLink]` above `keyup.enter` events could still propagate to parent expandable sections and toggle them. 
This fixes that issue (@artlowel 👍)